js/popups/tablemenupopup.ts

```diff
@@ -151,6 +151,10 @@ export class TableMenuPopup {
         this.addMoneyAllowed(currentTable.couldAddChips());
         this.handHistoryAllowed(playerIsInGame && currentTable.lastHandHistory() != null);
         this.leaveAllowed(myPlayer != null && !currentTable.myPlayerInGame() && myPlayer.IsSitoutStatus());
+        if (!appConfig.game.seatMode) {
```

Comment:
if it better to have 
```
if (appConfig.game.seatMode) {
this.leaveAllowed(…..)
} else {
this.leaveAllowed(true)
}`

or you could write without if like that
```
const leaveAllowed = myPlayer != null && !currentTable.myPlayerInGame() && myPlayer.IsSitoutStatus();
 this.leaveAllowed(!appConfig.game.seatMode || leaveAllowed );
```

I'm generally think it is bad to have "override" something for too long.